cmd/kpod/spec.go

```diff
@@ -17,6 +17,33 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func blockAccessToKernelFilesystems(config *createConfig, g *generate.Generator) {
+	if !config.privileged {
+		for _, mp := range []string{
+			"/proc/kcore",
+			"/proc/latency_stats",
```

Comment:
latency_stats isn't present in /proc on my RHEL PC.  If it's not present on what ever machine this is being run on, I think that's OK but just wanted to verify.

Reply:
Yup this is attempting to hide them, o them not existing is fine.